stratum/hal/lib/common/yang_parse_tree_test.cc

```diff
@@ -1002,6 +1002,50 @@ TEST_F(YangParseTreeTest,
   EXPECT_EQ(resp.update().update(0).val().string_val(), kMacAddressAsString);
 }
 
+// Check if the 'config/mac-address' OnUpdate action works correctly.
+TEST_F(YangParseTreeTest,
+       InterfacesInterfaceEthernetConfigMacAddressOnUpdateSuccess) {
+  auto path = GetPath("interfaces")(
+      "interface", "interface-1")("ethernet")("config")("mac-address")();
+  static constexpr char kMacAddressYangString[] = "11:22:33:44:55:66";
+  static constexpr uint64 kMacAddressUint64 = 0x112233445566ull;
+  static constexpr char kMacAddressSomethingString[] = "00;00;00;00;00;00";
```

Comment:
You used ; instead of : as separator, probably wrong?

Reply:
Yes, you're right.
But in this case, it's on purpose to test if a user inputs an unformed mac address, he/she will get a "wrong value" response.

Reply:
How about giving the malformed MAC string its own test (`InterfacesInterfaceEthernetConfigMacAddressOnUpdateFailure`)? There are more error cases (too long, too short, null bytes, ...) to be checked, which would clutter the success test case.